src/ansys/geometry/core/designer/body.py

```diff
@@ -358,6 +362,42 @@ def rotate(
         """
         return
 
+    @abstractmethod
+    def scale(
+        self,
+        value: Real,
+    ) -> None:
```

Comment:
I'd single line them if possible
```suggestion
    def scale(self, value: Real) -> None:
```